src/models/setup.model.ts

```diff
@@ -41,23 +41,24 @@ export enum SetupMigrationMode {
 }
 
 export enum Screen {
-  FINAL,
-  FORMAT,
-  INPUT_A,
-  INPUT_B,
-  INPUT_C,
-  INPUT_NODENAME,
-  LIGHTNING,
-  MIGRATION,
-  RECOVERY,
-  SETUP,
-  START_DONE,
-  SYNC,
-  WAIT,
+  FINAL = 0,
```

Comment:
Why is there a need to write it out explicitly?
https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
See comment below

Reply:
If it's the default and done automatically I'm fine with it :)

Reply:
This was created automaticaly: https://github.com/raspiblitz/raspiblitz-web/blob/7671bff849dbe316d91518616223ab0eb437527c/biome.json

I so far just modifier `ignore` and `indentStyle`